meta-mender-core/recipes-mender/mender-connect/mender-connect_1.2.0.bb

```diff
@@ -1,5 +1,7 @@
 require mender-connect.inc
 
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 2.5)"
```

Comment:
Very often, we just copy the old file. How is the best way to remember to remove this? A task set for Mender 3.2?

Reply:
Yes, I thought about that too. I am creating a reminder task, good idea!

Bu the way, I also considered to leave the `inc` with the dependency on `mender-client (>= 2.5)` and update only the master recipe... but decided for this approach which is more verbose but cleaner in my opinion.

Reply:
Created [MEN-5229](https://tracker.mender.io/browse/MEN-5229) :+1: